src/drivers/mct/main/seq_flux_mct.F90

```diff
@@ -1354,6 +1359,7 @@ subroutine seq_flux_atmocn_mct(infodata, tod, dt, a2x, o2x, xao)
        index_a2x_Sa_shum_HDO   = mct_aVect_indexRA(a2x,'Sa_shum_HDO', perrWith='quiet')
        index_a2x_Sa_shum_18O   = mct_aVect_indexRA(a2x,'Sa_shum_18O', perrWith='quiet')
        index_a2x_Sa_dens   = mct_aVect_indexRA(a2x,'Sa_dens')
+       index_a2x_Sa_uovern   = mct_aVect_indexRA(a2x,'Sa_uovern')
```

Comment:
How will this and the other changes in this module work for cesm?

Reply:
I have added in my recent commit conditional statement to make the changes relevant to E3SM only. Thanks!